desktop-bitcoin-wallet/index.html

```diff
@@ -143,7 +143,7 @@ <h1>Desktop Wallet</h1>
                     <img alt="download bitcoin desktop wallet" src="../img/mac-store-badge.svg">
                   </a>
                   <br />
-                  <a class="apk" href="/desktop-bitcoin-wallet/downloads/BlueWallet%205.6.60.dmg">
+                  <a class="apk" href="https://github.com/BlueWallet/BlueWallet/releases" target="_blank">
```

Comment:
I think this is bad for visit2download conversion. website should link to APKs and DMGs directly